README.md

```diff
@@ -1,6 +1,6 @@
-# Change In Christ
+# Sell Stuff
 
-Kyle Knight's blog
+A Platform to sell various things
 
 [![CircleCI](https://circleci.com/gh/WebJamApps/breakpointministries.svg?style=svg)](https://circleci.com/gh/WebJamApps/breakpointministries)
```

Comment:
this is not the correct link to circleci here, please get that setup and fix this please